vault/logical_system.go

```diff
@@ -1869,6 +1899,44 @@ func (b *SystemBackend) handleAuthTable(ctx context.Context, req *logical.Reques
 	return resp, nil
 }
 
+func augmentEnableAuthConfigMap(configMap map[string]interface{}) error {
+	configParamNameSlice := []string{
+		"audit_non_hmac_request_keys",
+		"audit_non_hmac_response_keys",
+		"passthrough_request_headers",
+		"allowed_response_headers",
+	}
+	for _, paramName := range configParamNameSlice {
+		if raw, ok := configMap[paramName]; ok {
+			outputSlice := []string{}
+
+			switch raw.(type) {
```

Comment:
This is perfectly fine, but you can make it even more succinct by assigning in the switch, e.g. : https://play.golang.org/p/x-RL2pAi1Vl

Reply:
Done